config.php

```diff
@@ -60,6 +60,9 @@
 define(BLOTTER_PATH, 'CHANGEME'); //Absolute html path to your blotter file, this feature is experimental and still is not fully functional.
 
 // Post & Thread
+define(EVENT_STICKY_RES, 1500); //The number of replies allowed to an event sticky before self-pruning begins. 
```

Comment:
Should be moved down to line 79/82 (nexzt to *MAX_RES*) for consistency.

Comment does not need to be multiple lines either.